templates/log4j2.xml.erb

```diff
@@ -138,7 +138,7 @@ if ! httpAppenders.empty?
 <%- end -%>
         </Logger>
 <%- if @herp -%>
-        <Logger name="org.openrepose.herp.post.filter" level="info">
+        <Logger name="org.openrepose.herp.post.filter" level="info" additivity="false">
```

Comment:
This is something that should have been in there all along.  Without this, repose needlessly logs all generated events to its log, which is nothing anyone would want to run with--only for debug.